### PR TITLE
Add BFloat16 support for cross, tril, triu, tril_indices, triu_indices and cumsum operators on CPU

### PR DESCRIPTION
**Pull request by CaoE**
Add BFloat16 support for cross, tril, triu, tril_indices, triu_indices and cumsum operators on CPU.

### CODE REVIEW

**Comment by ezyang** [comment]
@heitorschueroff do you think you can take care of this PR?

File: test/test_reductions.py [hunk 1]
```diff
@@ -416,6 +416,20 @@ def test_sum_integer_upcast(self, device):
         self._test_reduce_integer_upcast(lambda x, **kwargs: torch.sum(x, **kwargs), False)
         self._test_reduce_integer_upcast(lambda x, **kwargs: torch.sum(x, 0, **kwargs))
 
+    @onlyCPU
+    def test_cumsum_bfloat16(self, device):
```
Comment (heitorschueroff) [inline test/test_reductions.py:420]: There's also a test_cumsum in test_torch.py. Could you add BFloat16 dtype to those tests?

File: test/test_reductions.py [hunk 2]
```diff
@@ -416,6 +416,20 @@ def test_sum_integer_upcast(self, device):
...
+        input = torch.randn(5, 2, 8, 8, dtype=torch.float32, device=device).bfloat16().requires_grad_(True)
+        input2 = input.detach().clone().float().requires_grad_(True)
+        out = torch.cumsum(input,dim=0)
```
Comment (heitorschueroff) [inline test/test_reductions.py:423]: Can you please fix the lint issues

File: test/test_reductions.py (hunk 1 above)
Comment (CaoE) [reply test/test_reductions.py]: sure, I moved the BFloat16 test of cumsum to test_cumsum in test_torch.py.

File: test/test_reductions.py (hunk 2 above)
Comment (CaoE) [reply test/test_reductions.py]: fixed it.

**Review by heitorschueroff — APPROVED** [review]
LGTM! Thanks for the contribution.

**Pull request closed without merging**